src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,10 +4,18 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
+
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the unnecessary empty lines (3-7).